src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = new ColorSupplier().getRandomColor().toString();
+        int number = new Random().nextInt(100);
+        Ball ball = new Ball(color, number);
```

Comment:
same, let's make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method. And lets make number 100 as constant field https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names. And why do we need to call toString() method on getRandomColor() method in line 7?

Reply:
I can't explain about toString(). I hade many errors, and I did not consciously correct all of them